custom_components/plugwise/number.py

```diff
@@ -0,0 +1,104 @@
+"""Number platform for Plugwise integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.number import (
+    NumberEntity,
+    NumberEntityDescription,
+    NumberMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import COORDINATOR, DOMAIN, LOGGER
+from .coordinator import PlugwiseDataUpdateCoordinator
+from .entity import PlugwiseEntity
+
+
+@dataclass
+class PlugwiseEntityDescriptionMixin:
+    """Mixin values for Plugwse entities."""
+
+    command: str
+
+
+@dataclass
+class PlugwiseNumberEntityDescription(
+    NumberEntityDescription, PlugwiseEntityDescriptionMixin
+):
+    """Class describing Plugwise Number entities."""
+
```

Comment:
Why this empty class?

Reply:
Apparently this is how you create the top level class that recognizes the `command=` line.
This is brefa does it, and others in HA Core.